docs/vtex-io/Reference/concepts/vtex-io-documentation-builders/vtex-io-documentation-store-builder.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Store builder"
+slug: "vtex-io-documentation-store-builder"
+excerpt: "Learn how to use the VTEX IO Store builder."
+hidden: false
+createdAt: "2024-05-02T09:33:00.000Z"
+updatedAt: "2024-05-02T09:33:00.000Z"
+category: "App Development"
+---
+
+The `store` builder enables the development of [Store Framework](https://developers.vtex.com/docs/guides/vtex-io-documentation-what-is-vtex-store-framework) storefronts, empowering the development of both [storefront components](LINK) and unique store themes. The main use cases of the `store` builder include:
+
+- **Frontend apps**: When used alongside the [`react` builder](https://developers.vtex.com/docs/guides/vtex-io-documentation-react-builder), the `store` builder allows integrating a [frontend component](LINK) with Site Editor. It enables mapping a React component to a block, grouping store blocks, and defining route handling within a component.
```

Comment:
Mesma coisa
https://developers.vtex.com/docs/guides/vtex-io-documentation-1-developing-storefront-apps-using-react-and-vtex-io